docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb

```diff
@@ -0,0 +1,1316 @@
oclh3Ot/Jfw+NYrOTrFbIVYCLhvDTqwDj7xDrjYahUuNJKcyx8+Qr79rjwKqb7cNxhid5clQ2HDV6Xh8h34Fh0cBWOKfyt92IdzdevZ64CWlva+vIg5A/4bgYT9/o4I18yehfOvIZRGZmahO+g3hc9qXu6ictJywGa4V0uHr/vn+K/1v1R8eTS1efshtlR3Q9EF7PKPWLqnybzoPbSvE56qgq3ctjO/lO/enDwJGfeXtR06lXW6bU2q4rtuZkh3a76x0tV5QOIohn1KVPBPbzDzHx3ZPKaVdCjUZG26BQFW62yTshmtylJ554Aq2trXbHKysrce21Hc76jTfc2yCrpaUFR44cQWZmJrZt24aFCxfi+uuvR0tLi0vhzJs3D9XV1dZPaWmpW+nREtd3KzVIZzh7CnCiCKir5DeuAoDms3zjwhkGWHfasRM4oFjIlTYi++g2/ndX19WqWu/lJcdjNsuni4GjTk+IVL/+Tu07bMfPB5rrgZpjxt6fwVuoWAdn52uUdn1+fYR9ONuWeneNppqywfK9xWoaOgrnpGf090KCNEZIuzQPcseNugGRjP7p6eka+DiJT/a00evpWmlxORHWfodG8d+FwVrFtfH6NGxtfM3BDfzfukp+Ha0lHCj5lffZrr53nnWHg8VadRfWiKenp+vfvnIUnyOfrfgkVfjHBRv1hk9y0dZU1a+Ojivt4yBslDd+Pl9uLOFAeAJwtl1noXw5w6h+20PSe3Xr3HvduIgmnrCwsBDDhg2z2aTuv//9L3JycpCXl+dx+MnJyTCbzbjpJv6dqoMGDUKfPn1w5MgRlJeXWzv3HMehpKQEycnJsuFYLBZERETYfFjj0m6lzgzTkwpHuLZ8t7owhPdCr5rRsXFVdSnfIXDkJFx1LN6oROuq0LT2no7vSq/ScKfj6Inj9KbjMZBDP1K4RX1DRalRI60Aa44B/S4F7t2lf4fPCFTuBXoNBXKm8tMVZTaFUeVrlBphwrvsWW786FNvmeig073hRJPBFSf50sjve7wbuDd8sli/S5/jy2ziEHX6CZvMCX9dGazVNC+O759V97qqjld1xWcCs34E7trK57+mrKP8stp00VUMns7y797Qt5531q5w1WeLbXTXB/xfTsUsDJd8knd8qse+xtGMuuvf5ds4KaP48jP7RyC+fSrOlrfUlWWWg+LeonAtTHIPIbRqk/sgmnTqV61ahfvvvx8TJkzAU089hcmTJ+PFF1/E119/jUcffdTj8GNjYzF+/Hh8/fXXAIBDhw7h0KFDuOCCCzB48GC89957AICPPvoIvXr1kp16b1TMWo7ue1LhCNdW/aEyjHaHmJDTsVvnlOXOnYSrjkXrSrT9VRphp/9w/ioNdzqODvPnoDLxtuMxkEOXtXlXG/Fy67mvf7djfbW48cDy1VF6MWYuv75807/4fMq8BUCVr1FqhAlvGXCG21qraGixfkrnJqp9vBSjPl3wdHBFTb408vtuaw94zyeL9aut4MvshY+o06/+hO1ftWnUMi8q7p/ZbO6IU3hVV3UJ//3IT7b2YvDyyxaVa8TbfW5ozSF963ln7QpXfLbYRs8bD5w6wB//7D7guJP0q/VJXvSpHvkaxwHzf4U8HvmJ1+nMEb5ctZxTV5bVaqRXe0mLeCr3oj5+sGN7N1DbVw80W1N/00034fTp07j33nuRlJSEnTt3ols3ud0L3OPf//43br31Vvztb3+D2WzG4sWLkZiYiMWLFyM/Px/PPvssIiIisHTpUs3i1IPU1FR1JxphgxgBsWNsrAO+b1/bs3EBv0YnLEb5WtZr4tpfpdEW2RsB17YvB/nkLn79llRLdzYCUzrH2Tplb99f1rqLSExMaP9Pst+A7Do3laPqjhoPeu8EzAIVm8Ko8jWeTol0R2tP12cbHNU+XozRdlgX48ngis75ckt7AW/5ZHf1q6vq2CTr54VA6oXq06hVXlTev9TUVGDTx47jtMmvAWekyKJjOt1YIx5+5TNAaFjHb96u57VsVwg2GhYDbF8GwAQEWvhO6xsjgBF3AJcukL/WWf2/bSlfH333pNd8T2pqqnf3TRLy6G5ZVut39GovaRHPmLkIHtkEhITw311pk3fGfTigUae+pqYGd911F3bv3o1ffvkFa9aswZAhQ/Dmm2/ioosu0iIKpKamYsMG+7Uj6enp2LRpkyZxsKC4uFjdbqWqDNOTCqf9GutUJwfrzIVKvdcw4Gj76wN7DesY/XfkJF1u0Ghciba/SmOveKdSta/ScHddrZqGkLcdj1GectZVoeWzB2ABnHfgOnlnT3vkykrH/6p9jRRv3gcjd141wiXdhUZhY61xBnG1RHWDVBu/77bNAzo1BlXmU7rLfXUpX04u/yc/Q8dZGrXKi8r7V1xcjCy1cfqKn9cznW76xeIDB+Xt3dtPkbWIT7CXpe2dvGkfA90z+NdqvjcJKN/Fn+dqx1noPA6e4VWfWlxcjKxuLd7vEPtZJ9UhZrO8j1ex101nnSGkSa6ys7ORlJSEbdu2YcSIEXjhhRfw7rvv4s4778Qdd9yhRRSdFmHnfqc4M0xPphWJr133gOMwxFNZzhsH9L2Y/5x3ofbTWrwxVcpsBsxme93bj3sFNdN//MHxtE+vs9SVykwBbW/UblvCV8KaTn31ladAHiAuK5/fz38Am3Kj2teIcfk+KGitdsPDTjg1ziXdhQZo9hRj7x/g7tRJNVNANfT7btm8gLd9siv5FPbMiGzfLygymf9etU9dGrXKi9L9GzzDxh4CAwPVxanHelctpvnqvS7XzTXiHtk7awS7mP4Z8MAffIceALpn8t+nf8p/d3dpTl6+V32qbtp7va2oV3tJm3h82ua9gGZr6p999lkEBQVZj11wwQXYuXMnAgICtIii06K0qZ9LeFLhCNceL+TXlzed5f8e3yMfhrhSHzOXf8p90yreMWrpJL1ciWqiu1p8dIMvzWlvqDTd9j/bhsrR7R2N2x3L+fO06uwZdV2yloj
```

Comment:
I'd shorten this to "Plot the relation between the augmented features and the label" - otherwise it seems like you're always converting back-and-forth between temporian and pandas.